streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -1236,7 +1236,11 @@ public class StreamsConfig extends AbstractConfig {
                     Type.LONG,
                     null,
                     Importance.LOW,
-                    WINDOW_SIZE_MS_DOC);
+                    WINDOW_SIZE_MS_DOC)
+            .define(CONFIG_PROVIDERS_CONFIG, 
```

Comment:
We keep configs sorted by name, within each priority group. Please insert at the right place alphabetically.